docs/architecture/decisions/0002-dists-can-never-change-retroactively.md

```diff
@@ -0,0 +1,23 @@
+# 2. Distributions Can Never Change Retroactively
+
+**Date:** 2023-09-22
+
+## Status
+
+Accepted (for the GitHub package pipeline)
+
+## Context
+
+The Garden Linux apt repository produces daily distributions. 
+These distributions are assembled through a dedicated package repository pipeline. 
+Garden Linux maintainers define the package specifications for upcoming apt distributions.
+
+## Decision
+
+Apt distributions, identified by their version numbers, are immutable. They cannot be altered once established. Essentially, when a distribution is named with a specific version, its contents remain unchanged.
+
+## Consequences
+
+1. Introduction of the dynamic Garden Linux apt distribution named `next` 
```

Comment:
how is 'next' different from 'today'?

Reply:
today is like a soft link to the xyz.0 release. 

next is the development for "tomorrows" xyz.0 release